### DISCUSSION TITLE
How to remove the default padding/gap between elements on page?

### DISCUSSION THREAD

**Discussion by Anindya088** [zauberzeug/nicegui#2063] [Q&A] [answered]
### Question

Hi All,

I realized that when we write the elements one by one, they add some little gap by default. Is there a way to remove it?

Ex: If I just do this
```python
ui.column().classes('w-screen h-10 bg-blue-200')
ui.column().classes('w-screen h-10 bg-blue-200')
ui.column().classes('w-screen h-10 bg-blue-200')
```
it will look like this
![image](https://github.com/zauberzeug/nicegui/assets/65127974/30134ee8-894e-41a7-b6ea-15d359416a49)

Is there a way to remove that gap, like we can remove the page margins using the ```nicegui-content```?
I know we can do this by just nesting the elements under another column and then using ```gap-0```, but I was thinking if there is a general setting which we can use, without using more elements.

Please let me know if this is possible.

Thanks,
Anindya

**Comment by falkoschindler** [comment, marked as answer]
No, at the moment there is no general setting for customizing the gap.
Nesting a `ui.column` is a good solution, as well as setting the `gap` of the page `content`.
See https://github.com/zauberzeug/nicegui/discussions/1991#discussioncomment-7523293 for how to access the `content` in three different ways.

**Reply by Anindya088** [reply]
Hi @falkoschindler , thanks a lot. I just tried the ```ui.query('.nicegui-content').classes('p-0 gap-0')``` as you advised, and it works fine. So I can use this as an overall page setting :)

Thanks a lot,
Anindya
